test/run_test

```diff
@@ -934,7 +934,7 @@ run_pgaudit_test()
   # create a dir for config
   config_dir=$(mktemp -d --tmpdir pg-hook-volume.XXXXX)
   add_cleanup_command /bin/rm -rf "$config_dir"
-  -v DOCKER_EXTRA_ARGS || cp -r "$test_dir"/examples/pgaudit/* "$config_dir"
+  test -n "$DOCKER_EXTRA_ARGS" || cp -r "$test_dir"/examples/pgaudit/* "$config_dir"
```

Comment:
This means if the string is not empty. That's correct. Definition is here https://github.com/sclorg/postgresql-container/blob/master/test/run_test#L41. It means, that if variable is defined, then pgaudit is copied. Am I right?

Reply:
Otherwise LGTM.

Reply:
> This means if the string is not empty.

This is correct

> if variable is defined, then pgaudit is copied

Yes as long as the variable is empty, if it's not, it's not copied.